proxies/shared/resources/jsc/500DuplicateTemplates.ExtractDuplicates.js

```diff
@@ -1,3 +1,3 @@
 const body = context.getVariable("response.content");
-const message = JSON.parse(body).message;
+const {message} = JSON.parse(body);
```

Comment:
Does this syntax actually work in Apigee? Having a green pipeline should be a good proof that it's working but I'm not sure why it won't start.

Reply:
This syntax does work on apigee and it's making the pipeline fail as expected.